### PR TITLE
[Do not merge] Amplify Admin UI Storage integration docs

### PR DESCRIPTION
**Pull request by pananapread**
_Issue #, if available:_

_Description of changes:_
Adding docs for Amplify Admin UI Storage integration

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/pages/console/storage/file-browser.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+4. Select the file you want to move.
+5. Select the **Actions** dropdown and then select **Move**.
+6. Select or create the folder you want to move your to.
+7. Select **Move** to move your file to the selected folder.
+
+## To test your files
+We've provided several shortcuts to test your files in your code.
+
+### Copy S3 key
+You can copy your file's S3 key either by
```
Comment (siegerts) [inline src/pages/console/storage/file-browser.md:54]:
```suggestion
You can copy your file's S3 key either by:
```

File: src/pages/console/storage/file-browser.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+- Selecting **Actions**, and then **Copy S3 key**
```
Comment (siegerts) [inline src/pages/console/storage/file-browser.md:55]: Periods `.` needed here to match format above?

File: src/pages/console/storage/file-storage.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+5. On the **Storage** page, locate the **Authorization settings** on the bottom of the *File storage bucket* card.
+6. Select your desired authorization settings for **Signed-in users**, or **Guest users** _(optional)_.
+
+The authorization modes can always be edited under **Set up**, **Storage**.
+
+## To import file storage
+
+For default file access levels to work, your bucket needs to be configured accordingly:
+
+public/ - Accessible by all users of your app
```
Comment (siegerts) [inline src/pages/console/storage/file-storage.md:29]: Should these bucket paths follow the  unordered, bolded format from above?

**Review by siegerts — COMMENTED** [review]
A few format notes

**Review by siegerts — APPROVED** [review]

**Review by aspittel — APPROVED** [review]
LGTM!

**Comment by jakeburden** [comment]
Hey! Heads up the file extension should be `.mdx` and for the pages to show in the sidebar they'll need to be added to this config:

https://github.com/aws-amplify/docs/blob/main/src/directory/directory.js#L1584

**Comment by jakeburden** [comment]
Staging URL: https://pananapread-main.d1oosfztpocl9c.amplifyapp.com

**Pull request merged**